lib/solidus_dev_support/templates/extension/bin/sandbox.tt

```diff
@@ -69,4 +77,5 @@ unbundled bundle exec rails generate solidus:auth:install
 
 echo
 echo "🚀 Sandbox app successfully created for $extension_name!"
+echo "🚀 Using $RAILSDB and Solidus $BRANCH"
```

Comment:
Should mention `$SOLIDUS_BRANCH` instead of just `$BRANCH` that is local to the sandbox script.

Reply:
it's actually the `$BRANCH`, the `$SOLIDUS_BRANCH` is what I check for if present and assign that to `$BRANCH` or set it to `'master'` if not present.

Reply:
Sure, I just re-read the code to be certain 😅, so you're right that `$BRANCH` is the recipient of the final value, but what I meant was that for the user the way that value can be controlled is through the `$SOLIDUS_BRANCH` env var, e.g.:

```
$ env BRANCH=foo bin/sandbox # => will still reassign "master" to $BRANCH
$ env SOLIDUS_BRANCH=v2.9 bin/sandbox # => will assign $BRANCH with "v2.9" 
```

So since those messages are a sort of inline documentation they should hint the user about variable that can affect how the sandbox is generated. The same (I just noticed) goes for `$RAILSDB` vs. `$DB`.

So it might look like:

```bash
echo "🚀 Use `export DB=[postgres|mysql|sqlite]` to control the DB adapter"
echo "🚀 Use `export SOLIDUS_BRANCH=<BRANCH-NAME>` to control the Solidus version"
```

Reply:
Ah! Yes documentation like that.. I like it. Will update the README too. Commit is coming in a few